src/test/java/net/minecraftforge/debug/KnockBackHookTest.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.entity.passive.EntityCow;
+import net.minecraft.entity.passive.EntitySheep;
+import net.minecraftforge.event.entity.living.LivingKnockBackEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid = "kbhtest", name = "Knock Back Hook Test", version = "1.0")
+@Mod.EventBusSubscriber
+public class KnockBackHookTest
+{
+	private static final boolean ENABLED = false;
```

Comment:
Can you put 4 spaces for tabs?

Reply:
As liach says, please use spaces for indentation.